app/controllers/subscriptions_controller.rb

```diff
@@ -79,6 +79,8 @@ def govuk_url
       sanitised_referer_uri.query = referer_uri.query
       sanitised_referer_uri.fragment = referer_uri.fragment
       sanitised_referer_uri.to_s
+    elsif @subscriber_list["url"].match?(%r{^/get-ready-brexit-check/results})
```

Comment:
I'm wondering if we even need this check because right now only subscriber lists from the tool will have such URLs (and in the future we might want to support any subscriber list with a URL supporting the back button)... but not a big deal if we want to get this out.

Reply:
Yup true, when we think of an actual solution to the main problem this can be edited accordingly.

Reply:
And having the original page url on all subscriber lists would improve the journey but still isn't great as really the back button should go to the previous page and not shoot the user to the original page which would skip a few pages in the signup flow, like the confirm and subscribe pages for example.